backend/app/store/search/service/search_service.go

```diff
@@ -0,0 +1,89 @@
+package search
```

Comment:
This should be in the root folder, otherwise project won't compile:
```
app/store/service/service.go:40:25: undefined: "github.com/umputun/remark42/backend/app/store/search".Service
```